### DISCUSSION TITLE
如何去掉载入动画？

### DISCUSSION THREAD

**Discussion by kulkatt** [hooke007/mpv_PlayKit#457] [Q&A 问答]
![image](https://github.com/hooke007/MPV_lazy/assets/24421888/846a343b-bcf3-48e5-a3dc-9f01befea62b)
就是这个符号，在读盘和下载字幕的时候会出现，经常会干扰画面。。。

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/0b0e0a6d49fa67edccdb0f3f7fea5f8b432b15fa/portable_config/script-opts.conf#L581-L583

**Reply by kulkatt** [reply]
谢谢，
试了在script-opts.conf中添加
script-opts-append = uosc-disable_elements= buffering_indicator,pause_indicator 
仍然还是会显示
![image](https://github.com/hooke007/MPV_lazy/assets/24421888/34d0cece-3764-4bd5-a9ba-fdee7124f20f)
但是类似YouTube那个暂停/继续图标是正常消失掉了

**Reply by hooke007** [reply]
> 用鼠标按住拖动进度条的时候还是会有这个符号

什么符号

**Comment by kulkatt** [comment]
> > 用鼠标按住拖动进度条的时候还是会有这个符号
> 
> 什么符号
![image](https://github.com/hooke007/MPV_lazy/assets/24421888/4ba7c0f9-ad12-48f1-9acf-b4f32c105cb6)
试了下buffering_indicator选项貌似没用, 但是pause_indicator 有用
